packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.html

```diff
@@ -84,8 +84,8 @@
                 <dl class="oui-tile__definition">
                     <dt class="oui-tile__term">
                         <span data-translate="kube_service_file"></span>
-                        <oui-popover>
-                            <button type="button" class="oui-popover-button" oui-popover-trigger></button>
+                        <oui-popover data-oui-popover-placement="auto">
```

Comment:
Why is not the default behavior ? :(

Reply:
Right now, the default behavior is right. Will discuss with Axel on this :)

Reply:
The default behaviour was asked `top-center` in the guidelines. Need to see how reacts Popper.js, but we could set it to default `auto` for the next major update (v4).

But here, the preoccupation should be that we should not use `oui-popover` component anymore (deprecated) :p
It should be used as a directive, to help for the migration.